packages/server/lib/cloud/api/index.ts

```diff
@@ -227,6 +232,15 @@ const isRetriableError = (err) => {
     (err.statusCode == null)
 }
 
+function noproxyPreflightTimeout (): number {
+  try {
+    return !_.isUndefined(process.env.CYPRESS_INTERNAL_INITIAL_PREFLIGHT_TIMEOUT) ?
```

Comment:
Is this not intended for users to set? CYPRESS_INTERNAL is intended to never be used by users.